sydtest/src/Test/Syd/Output.hs

```diff
@@ -463,8 +469,36 @@ splitChunksIntoLines =
 
 outputEqualityAssertionFailed :: String -> String -> [[Chunk]]
 outputEqualityAssertionFailed actual expected =
-  let diff = V.toList $ getTextDiff (T.pack actual) (T.pack expected)
-      -- Add a header to a list of lines of chunks
+  let diffM = unsafePerformIO $ do
+        let diff = getTextDiff (T.pack actual) (T.pack expected)
+        -- we want to evaluate the diff in order to ensure that its
+        -- computation happen in the timeout block
+        -- and is not instead lazyly passed to the caller.
+        --
+        -- The safe option would be to @force diff@ to normal form, however
+        -- there are no 'NFData' instances on 'Diff Text'.
```

Comment:
I own that code so we can go put them there.

Reply:
https://github.com/NorfairKing/fast-myers-diff/blob/6cee293f3f32c443295e589330cf30a46ada0045/fast-myers-diff/src/Myers/Diff.hs#L52

Reply:
https://github.com/NorfairKing/fast-myers-diff/pull/1 MR is here, I let you merge (if you accept) and I'll rework the current MR here.

(I'm really not a friend of hackage bounds, but looks like a release of `fasty-myers-diff` will be necessary in order for `sydtest` to be released with new bounds)

Reply:
MR upgraded, it now uses `force` and `fast-myres-diff > 0.0.1`.